mmv1/products/networkservices/TcpRoute.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'TcpRoute'
+base_url: 'projects/{{project}}/locations/global/tcpRoutes'
+create_url: 'projects/{{project}}/locations/global/tcpRoutes?tcpRouteId={{name}}'
+self_link: 'projects/{{project}}/locations/global/tcpRoutes/{{name}}'
+min_version: beta
+update_verb: :PATCH
+update_mask: true
+description: |
+  TcpRoute is the resource defining how TCP traffic should be routed by a Mesh/Gateway resource.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1beta1/projects.locations.tcpRoutes'
```

Comment:
Is the user guide available to add here?

Reply:
Done